src/components/ReactSlider/ReactSlider.md

```diff
@@ -114,9 +114,9 @@ Track changes with `onBeforeChange`, `onChange`, and `onAfterChange` event handl
     className="horizontal-slider"
     thumbClassName="example-thumb"
     trackClassName="example-track"
-    onBeforeChange={val => console.log('onBeforeChange value:', val)}
-    onChange={val => console.log('onChange value:', val)}
-    onAfterChange={val => console.log('onAfterChange value:', val)}
+    onBeforeChange={(val,index) => console.log(`onBeforeChange: index: ${index} value:${val}`)}
+    onChange={(val,index) => console.log(`onChange: index: ${index} value:${val}`)}
+    onAfterChange={(val,index) => console.log(`onAfterChange: index: ${index} value:${val}`)}
```

Comment:
Can we clean up the logging a little bit in these examples? There is some inconsistent spacing and the arguments are logged in a different order than the order in which they are found in the function.

Maybe something like this would clean things up?

```jsx
    onBeforeChange={(value, index) => console.log(`onBeforeChange: ${JSON.stringify({ value, index })}`)}
    onChange={(value, index) => console.log(`onChange: ${JSON.stringify({ value, index })}`)}
    onAfterChange={(value, index) => console.log(`onAfterChange: ${JSON.stringify({ value, index })}`)}
```

Reply:
done